catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/utils/BrickSpinnerDataInteractionWrapper.java

```diff
@@ -52,6 +55,16 @@ public BrickSpinnerDataInteractionWrapper checkShowsText(String text) {
 		return new BrickSpinnerDataInteractionWrapper(dataInteraction);
 	}
 
+	public BrickSpinnerDataInteractionWrapper checkHasVariable(String variableName) {
+		dataInteraction.perform(click());
+		onView(allOf(
+				withText(variableName),
+				hasSibling(withText(R.string.brick_variable_spinner_create_new_variable)))).
```

Comment:
why is hasSibling necessary?
if this is actually checking if a string is there, and takes a string as parameter,  do not name is checkHas**Variable**
please just make a new version of checkValuesAvailable that takes an array of strings, or implement a new method just taking one string and call it "checkContains"